gdrive/export_api.py

```diff
@@ -153,3 +153,47 @@ async def find(find: FindModel):
     )
     export_data = export_client.find(responseId, find.field, find.values, result)
     return export_data
+
+
+# ------------------------------- Archive API --------------------------------------
+
+
+class InteractionModel(BaseModel):
+    interactionId: str
+    driveId: str
+
+
+@router.post("/export/vendor-response-list")
+async def get_vendor_responses(request: InteractionModel):
+    """
+    Returns a list of Google Drive object IDs that contain the
+    vendor responses for this particular interaction
+    """
+    interaction_folders = drive_client.get_files_by_drive_id(
+        filename=request.interactionId, drive_id=request.driveId
+    )
+    vendor_file_ids = []
+    for dir in interaction_folders:
+        files = drive_client.get_files_in_folder(id=dir["id"])
+        for file in files:
+            if file["mimeType"] == "application/json" and (
```

Comment:
Hmm. I think just return all here and move this logic to consumer.  Rename this to something like getfilesforId